recipes/jupyter_contrib_nbextensions/post-link.bat

```diff
@@ -0,0 +1,5 @@
+@echo off
+:: We redirect stderr & stdout to conda's .messages.txt; for details, see
+::     http://conda.pydata.org/docs/building/build-scripts.html
+"%PREFIX%\Scripts\jupyter.exe" contrib nbextension install --sys-prefix >> "%PREFIX%/.messages.txt" 2>&1
```

Comment:
no, that is how it's meant to be called, and it works in CI testing (see, e.g. https://travis-ci.org/ipython-contrib/jupyter_contrib_nbextensions/jobs/148365017#L395 from https://github.com/ipython-contrib/jupyter_contrib_nbextensions/blob/master/tests/test_application.py#L135